### PR TITLE
Issue #2350 Update timeout of running integration test

### LINKED ISSUE #2350: Update timeout of running integration test
Currently, the timeout is 2hr 

https://github.com/minishift/minishift/blob/82886edd25d24b1bff44d8246f5aad24d9040860/Makefile#L35

as it is the root cause of nightly job failures
https://ci.centos.org/job/minishift-nightly-b2d/143/consoleFull

### PR DESCRIPTION
**Pull request by budhrg**
Fix #2350

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,7 +32,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
-TIMEOUT ?= 7200s
+TIMEOUT ?= 10800s
```
Comment (amitkrout) [inline Makefile]: +1
3hrs quite enough atm

Comment (praveenkumar) [reply Makefile]: +1

Comment (agajdosi) [reply Makefile]: +1

**Comment by budhrg** [comment]
retest this please

File: centos_ci.sh [hunk 2]
```diff
@@ -367,7 +367,7 @@ else
     perform_docs_publish "$REPO_OWNER" "$BRANCH" "minishift/docs/ondemand/$REPO_OWNER-$BRANCH";
   elif [[ "$JOB_NAME" = "minishift-release" ]]; then
     perform_release $RSYNC_PASSWORD;
-  elif [[ "$JOB_NAME" = "minishift-nightly-b2d" ]]; then
+  elif [[ "$JOB_NAME" = "minishift-pr" ]]; then
```
Comment (LalatenduMohanty) [inline centos_ci.sh]: This needs to changed back before merging right?

Comment (budhrg) [reply centos_ci.sh]: Yes @LalatenduMohanty

File: test/integration/features/cmd-openshift.feature [hunk 3]
```diff
@@ -103,8 +103,8 @@ cluster in VM provided by Minishift.
 
   Scenario: Deployed Node.js application is available
       When service "nodejs-ex" rollout successfully within "1200" seconds
-      Then with up to "5" retries with wait period of "1000ms" the "body" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" contains "Welcome to your Node.js application on OpenShift"
-       And with up to "5" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" is equal to "200"
+      Then with up to "30" retries with wait period of "1000ms" the "body" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" contains "Welcome to your Node.js application on OpenShift"
```
Comment (LalatenduMohanty) [inline test/integration/features/cmd-openshift.feature]: @minishift/minishift-dev  FYI , we are going for 30 retries now.

**Pull request merged**